File: s5/1934/1934.py
t = int(input())
result = []
def gcd(a, b):
	r = a%b
	if r==0:
		return b
	else:
		return gcd(b, r)
for i in range(t):
	a, b = map(int, input().split())
	result.append(a*b//gcd(a,b))
for i in result:
	print(i)

File: s5/1934/1934.cpp
#include<cstdio>

int gcd(int a, int b)
{
	int r = a%b;
	if(r==0)
		return b;
	else 
		return gcd(b, r);
}

int main()
{
	int t;
	scanf("%d", &t);
	int *result = new int [t];
	for(int i=0; i<t; i++)
	{
		int a, b;
		scanf("%d %d", &a, &b);
		result[i] = a*b/gcd(a, b);
	}
	for(int i=0; i<t; i++)
		printf("%d\n", result[i]);

	delete[] result;
	return 0;
}
